bin/copy-labels.js

```diff
@@ -0,0 +1,322 @@
+#!/usr/bin/env node
+
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const util = require('util');
+
+const { LabelHandler } = require('../lib/labels.js');
+
+const toolName = path.basename(process.argv[1]);
+const toolVersion = require('../package.json').version;
+
+const help = `\
+This tool copies labels from source repository or json file to destination
+repository or json file.
+
+Usage: ${toolName} [OPTION]... SOURCE DEST
+       ${toolName} --help
+       ${toolName} --version
+
+Options:
+  --config  path to the config file
+  --user    GitHub username
+  --token   GitHub access token
+  --delete  delete unused labels in destination repository before copy
+  --update  update existing labels with the same name
+  --dry-run print operations that will be executed and exit
+  --help    print this help message and exit
+  --version print version and exit
+`;
+
+const options = new Map([
+  ['--config', 'config'],
+  ['--user', 'user'],
+  ['--token', 'token'],
+  ['--delete', 'delete'],
+  ['--update', 'update'],
+  ['--dry-run', 'dry'],
+]);
+
+const helpExit = (message, error) => {
+  console.error(message);
+  if (error) console.error(error.message);
+  console.error(`\nTry '${toolName} --help' for more information`);
+  process.exit(1);
+};
+
+const getArgs = () => {
```

Comment:
```suggestion
const processArgs = () => {
```
Or maybe `handleArgs` considering this also shows help/version and exits in those cases.